src/emqx_acl_cache.erl

```diff
@@ -145,6 +148,16 @@ foreach_acl_cache(Fun) ->
     _ = map_acl_cache(Fun),
     ok.
 
+%% All acl cache entries added before `start_cache_drain()` invocation will become expired
+start_cache_drain() ->
+    _ = persistent_term:put(drain_k(), time_now()),
+    ok.
+
+%% Stops discarding acl cache started with `start_cache_drain()`
+stop_cache_drain() ->
```

Comment:
i think this function does absolutely nothing as when `start_cache_drain()` is invoked all previous cache items disappear, items added after that stay no matter `stop_cache_drain()` is invoked or not.

i propose renaming `start_cache_drain` to `drain_cache` and remove `stop_cache_drain` completely

Reply:
agree.